lib/config/store/pre-auth-hook.js

```diff
@@ -39,25 +39,25 @@ function onStorePreAuth (request, reply) {
     }
 
     return server.plugins.account.api.sessions.find(sessionToken)
-  })
 
-  .then(function (session) {
-    delete request.headers.authorization
-    request.headers.cookie = 'AuthSession=' + session.id
+    .then(function (session) {
```

Comment:
IMHO, adding `.then()` clauses to a Promise that is then returned increases the indent level unnecessarily, reducing readibility